ons_alpha/jinja2/templates/components/streamfield/ons_table_block.html

```diff
@@ -0,0 +1,18 @@
+<div class="ons-table-block">
+    {% if value.heading %}
+        <h3>{{ value.heading }}</h3>
+    {% endif %}
+
+    {{ table }}
+
+    {% if value.source %}
+        <p><strong>Source:</strong> {{ value.source }}</p>
+    {% endif %}
+
+    {% if value.footnotes %}
+        {% from "components/details/_macro.njk" import onsDetails %}
```

Comment:
note: this aligns the footnotes with the bulletin prototypes shown at this week's Show & Tell - https://digital-publishing-prototypes.netlify.app/prototypes/new-nav/retail-sales-bulletin